src/core/environment.go

```diff
@@ -510,22 +510,82 @@ func processors() (res []*proto.CpuInfo) {
 }
 
 func processorCache(item cpu.InfoStat) map[string]string {
+	cache := getProcessorCacheInfo(cpuid.CPU)
+	if cpuid.CPU.Supports(cpuid.SSE, cpuid.SSE2) {
+		cache["SIMD 2:"] = "Streaming SIMD 2 Extensions"
+	}
+	return cache
+}
+
+func getProcessorCacheInfo(cpuInfo cpuid.CPUInfo) map[string]string {
+	cache := getDefaultProcessorCacheInfo(cpuInfo)
+
+	out, err := exec.Command("lscpu").Output()
+	if err != nil {
+		log.Warnf("Error executing lscpu on host: %v", err)
```

Comment:
can you update the warning to say please install lscpu in order to collect processor cache information - this is not installed on FreeBSD by default

Reply:
If you can test the lscpu output on FreeBSD and the code works with it, I think this PR is good then

Reply:
@oliveromahony At the moment there is a bug where Nginx agent could not be installed on FreeBsd. 
Please suggest on how should I proceed with testing?